WooFoundation/WooFoundation/Colors/ColorPalette.xcassets/WooCommercePurple0.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0xFF",
-          "green" : "0xED",
-          "red" : "0xF2"
+          "blue" : "255",
+          "green" : "237",
+          "red" : "242"
```

Comment:
I used decimal values instead of hex here because this was easier (the decimal values were already provided in the Figma file), but I also believe this is better as it ensures what we have matches exactly the source file (bYRUD9txDt1tvVmZifswdU-fi-5050_19256).